### DISCUSSION TITLE
Under review

### DISCUSSION THREAD

**Discussion by JaanakiSwaroop** [github/india#657] [Students]
I have registered, and it's been 5 days long but the application is still under review. Please check it out

**Comment by ayushgarg890** [comment]
My application is also under review. waiting for  confirmation

**Comment by aditi2906** [comment]
I am facing the same issue...I had timely registered for the event still my profile is under review. Please help!

**Comment by Prathyusha-Guduru** [comment]
Yes, please look into it! its been 8 days since I registered, I even authenticated with my college mail id and id card, dont know what went wrong.

**Comment by tancoder24** [comment]
I have registered, and it's been 5 days long but the application is still under review. Please check it out

**Comment by skmodi649** [comment]
I had registered on the opening day itself, I have even provided college mail Id and the id card details as well, but still it is under review

**Reply by Konaparthy-Niharika** [reply]
Even my profile is under review since  6 days.. I am not understanding what is happening

**Reply by Rajatino16** [reply]
I am also facing the same issue.

**Comment by naveeramesh** [comment]
My profile is under review more than 2 weeks please take into consideration.

**Comment by YashwanthAlluri** [comment]
It's been more than a week and my application is still under review.
